src/app/shopping-list/shopping-list/shopping-list.component.ts

```diff
@@ -134,8 +135,15 @@ export class ShoppingListComponent implements OnInit, AfterViewInit, OnDestroy {
 	}
 
 	public editIngredient(ingredient: IngredientsInStore): void {
-		this.editingIngredientId = ingredient.id;
-		this.ingredientForm.patchValue(ingredient);
+		this.editingIngredient = ingredient;
```

Comment:
`editingIngredient` sounds like boolean. IMO better name would be `editedIngredient`